integration-tests/contract-originate-contract-with-sapling.spec.ts

```diff
@@ -4,17 +4,16 @@ import { Protocols } from "@taquito/taquito";
 
 CONFIGS().forEach(({ lib, rpc, protocol, setup,  }) => {
   const Tezos = lib;
+  const edonet = (protocol === Protocols.PtEdo2Zk) ? require('jest-retries') : test.skip;
 
-  const edonet = (protocol === Protocols.PtEdo2Zk) ? test && require('jest-retries') : test.skip;
-  
   describe(`Test origination of contracts with sapling using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup();
       done()
     })
-
-     edonet('Originates a contract having an empty sapling state in its storage', 2, async (done: () => void) => {
+    //  deepcode ignore only-arrow-functions: not sure how to do it otherwise
```

Comment:
Do you think this line can be removed if the following one is written this way: 
edonet('Originates a contract having an empty sapling state in its storage', 2, async (done: () => void) => {

Reply:
i will change it

Reply:
made the change